### ISSUE TITLE
Splat route does not catch root index.tsx

### ISSUE THREAD

**Issue by KasparRosin** [remix-run/remix#3814]
### What version of Remix are you using?

1.6.3

### Steps to Reproduce

create the following route structure, no index.jsx file.

```
- routes/
  - $.jsx
  ```

  Navigate to localhost:3000

### Expected Behavior

You can see the insides of $.jsx file.

### Actual Behavior

You are shown an blank page.

**Comment by KasparRosin** [comment]
This is an issue due to us having locales in front of the urls.

/ <- Home
/en <- Home in english
/ru <- Home in russian 

you get the point.
That's why I don't want to create

```
- routes/
  - index.jsx
  - lang/
     - $.jsx

**Comment by KasparRosin** [comment]
![image](https://user-images.githubusercontent.com/33309407/180224240-4d533490-e6e7-460d-8deb-71b7a1d259e5.png)
Solved it currently by doing this.
Changing remix.config.js file did not help, since one file can only be assigned to one route.

**Comment by machour** [comment]
Closing as this seems to be fixed.
By the way, it's splaT, not splaSH route <3

**Comment by devarshiqm** [comment]
@machour this issue still exist, can we please have more information i.e where it was fixed, since I can reproduce it in `1.13.0`
